proto/sentry_protos/snuba/v1/examples.py

```diff
@@ -263,3 +272,165 @@ def test_example_table_with_aggregations() -> None:
             offset=2
         ),  # if we're ordering by aggregate values, we can't paginate by anything except offset
     )
+
+
+def test_example_find_traces() -> None:
+    # Find traces that contain a span event with a `span_name` of "database_query"
+    FindTracesRequest(
+        meta=COMMON_META,
+        filter=TraceFilter(
+            event_filter=EventFilter(
+                trace_item_name="spans",
+                filter=TraceItemFilter(
+                    comparison_filter=ComparisonFilter(
+                        key=AttributeKey(
+                            type=AttributeKey.TYPE_STRING,
+                            name="sentry.span_name",
+                        ),
+                        op=ComparisonFilter.OP_EQUALS,
+                        value=AttributeValue(val_str="database_query"),
+                    ),
+                ),
+            ),
+        ),
+    )
+
+    # Find traces with a single span event with a `span_name` of "database_query"
+    # and a `transaction_name` of "GET /v1/rpc"
+    FindTracesRequest(
+        meta=COMMON_META,
+        filter=TraceFilter(
+            event_filter=EventFilter(
+                trace_item_name="spans",
+                filter=TraceItemFilter(
+                    and_filter=AndFilter(
+                        filters=[
+                            ComparisonFilter(
+                                key=AttributeKey(
+                                    type=AttributeKey.TYPE_STRING,
+                                    name="sentry.span_name",
+                                ),
+                                op=ComparisonFilter.OP_EQUALS,
+                                value=AttributeValue(val_str="database_query"),
+                            ),
+                            ComparisonFilter(
+                                key=AttributeKey(
+                                    type=AttributeKey.TYPE_STRING,
+                                    name="sentry.transaction_name",
+                                ),
+                                op=ComparisonFilter.OP_EQUALS,
+                                value=AttributeValue(val_str="GET /v1/rpc"),
+                            ),
+                        ]
+                    ),
+                ),
+            ),
+        ),
+    )
+
+    # Find traces that contain two events: a span with a `span_name` of
+    # "database_query" and an error with a `group_id` of "1123"
```

Comment:
Having 2 trace filters for 2 different spans in the same trace is a valid use case as well right?

Reply:
Yep!